server/config/summaries/place_summaries.json

```diff
@@ -0,0 +1,152 @@
+{
+ "geoId/3651000": {
+  "summary": "New York City, NY is the most populous city in the United States. The population of New York City has fluctuated between 7.3 million and 8.8 million people between 1990 and 2021. New York City has the 2nd highest median income in Staten Island, Queens, Manhattan, Brooklyn, and The Bronx. The median individual income in New York City has increased from $27,252 in 2011 to $36,302 in 2021. The median age in New York City has increased from 35.4 in 2011 to 37.3 in 2021."
```

Comment:
the sentence " New York City has the 2nd highest median income in Staten Island, Queens, Manhattan, Brooklyn, and The Bronx." doesn't make sense

Reply:
it has the second highest among cities in those boroughs. is it the phrasing of the sentence?